.github/CODEOWNERS

```diff
@@ -663,6 +663,7 @@ peps/pep-0781.rst  @methane
 peps/pep-0782.rst  @vstinner
 peps/pep-0783.rst  @hoodmane @ambv
 peps/pep-0784.rst  @gpshead
+peps/pep-0785.rst  @gpshead
```

Comment:
adding a dummy comment to resolve to trick github into recalculating its :github-on-fire: state.